app/presenters/claim_csv_presenter.rb

```diff
@@ -11,7 +11,10 @@ def journeys
   end
 
   def sorted_and_filtered_state_transitions
-    claim_state_transitions.sort.reject { |transition| (transition.to == 'draft' || transition.to == 'archived_pending_delete') }
+    claim_state_transitions.sort.reject do |transition|
+      %w(draft archived_pending_delete).include?(transition.to) ||
+        transition.created_at < Time.now - 6.months
```

Comment:
When we move this out, it should probably be configurable somewhere – but for now, fair enough.